redis/commands/bf/commands.py

```diff
@@ -416,6 +417,16 @@ def info(self, key):
         """  # noqa
         return self.execute_command(TDIGEST_INFO, key)
 
+    def trimmed_mean(self, key, low_cut_quantile, high_cut_quantile):
```

Comment:
I think this function need to be in the `TDigestCommands` class

Reply:
It is already in `TDigestCommands` class

Reply:
Sorry, my mistake... Ignore it